### ISSUE TITLE
Unpoly breaks scroll behavior on page refresh (resets scroll position to top of the page)

### ISSUE THREAD

**Issue by alexandergitter** [unpoly/unpoly#366]
### Bug description

Normally when hitting the browser's refresh button, browsers restore the scroll position from before the refresh (you can try that on pretty much any website, e.g. https://www.nytimes.com/).
With Unpoly, this behavior breaks - presumably because it sets `window.history.scrollRestoration = 'manual'`.

### Reproduction project

https://demo.unpoly.com/

### Steps to reproduce the behavior:

1. Scroll to the bottom of the page 
2. Hit F5 / the browser's refresh button
3. Note that unlike standard browser behavior, the scroll position has been reset to the top of the page

### Expected behavior

Refreshing the page should behave like the standard browser behavior and keep the original scroll position.

### Browser version

 - OS: Ubuntu Linux 20.04
 - Browser Chromium, Firefox

### Additional info

I'm sure there's good reason to set `scrollRestoration` to `manual` when working with the history API. In my case I don't use Unpoly for full page navigation and I workaround this unwanted behavior with the following snippet:
```
window.addEventListener("pagehide", () => {
  window.history.scrollRestoration = "auto";
});
```

**Comment by triskweline** [comment]
> I'm sure there's good reason to set scrollRestoration to manual when working with the history API.

I don't remember if there was a good reason. If it doesn't break tests we can just remove it.

**Comment by triskweline** [comment]
Fixed in version 2.6.0.
